src/context/internal.rs

```diff
@@ -38,15 +51,21 @@ impl Isolate {
         unsafe { std::mem::transmute(ptr) }
     }
 
-    pub(crate) fn current() -> Isolate {
+    #[cfg(feature = "napi-runtime")]
+    pub(crate) fn current() -> Env {
+        panic!("Context::current() will not implemented with n-api")
```

Comment:
This method is impossible with `N-API`--there isn't an equivalent to `Isolate->GetCurrent()`. It will be the source of most breaking changes.

Reply:
Yes, but, there are a bunch of places that it needs to be refactored out of (`src/context/mod.rs`) before it can be removed. I think it's better to leave that as a future PR.